aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -453,6 +453,44 @@ The default value is 96 KB (98,304).
 
 ::: moniker-end
 
+### Synchronous IO
```

Comment:
Needs a warning: A large number of blocking synchronous IO operations can lead to thread pool starvation and make your application unresponsive. Only enable this if you're using a library that does not support async IO.